pkg/config/testdata/tnf_test_config.yml

```diff
@@ -0,0 +1,43 @@
+generic:
+  containersUnderTest:
+    - namespace: default
+      podName: test
+      containerName: test
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.8
+    - namespace: default
+      podName: partner
+      containerName: partner
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.29
+  partnerContainers:
+    - namespace: default
+      podName: partner
+      containerName: partner
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.29
+  testOrchestrator:
+    namespace: default
+    podName: partner
+    containerName: partner
+operators:
+  - name: etcdoperator.v0.9.4
+    namespace: default
+    autogenerate: false
+    tests:
+      - OPERATOR_STATUS
+    certifiedoperatorrequestinfo:
+      - name: "etcd-operator"
+        organization: "redhat-marketplace"
```

Comment:
I want to ensure that this is not actually reaching out to the API from the unit test context?  If so, then if that service falls over, our unit tests will perpetually fail.

Reply:
Not at all, this config is not used for anything except to load a file and check that (some) of it is loaded successfully.
I had exactly the same concern with the tests for that spec itself.